config/default/manager_image_patch.yaml

```diff
@@ -10,3 +10,4 @@ spec:
       # Change the value of image field below to your controller image URL
       - image: kfserving-controller:latest
         name: manager
+        imagePullPolicy: IfNotPresent
```

Comment:
for real deployments we may want `PullAlways` though

Reply:
I have added a development kustomize overlay

Reply:
Don't we want PullAlways for the devloop too?

Reply:
For local minikube testing I build the image in minikube so don't want to pull from anywhere. For external development yes I suppose you do. Maybe if we have a preferred development guide we could sync on what is best practise. Happy to deploy internal docker repo to cluster as well in which case always pulling makes sense. Shall we merge this PR and then update with advice from @yuzisun and yourself?